go.mod

```diff
@@ -26,19 +26,48 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.4.0
-	github.com/vishvananda/netlink v1.0.0
+	github.com/vishvananda/netlink v1.1.0
 	golang.org/x/net v0.0.0-20201202161906-c7110b5ffcbb // indirect
 	golang.org/x/text v0.3.4 // indirect
 	gopkg.in/yaml.v2 v2.4.0
 	k8s.io/api v0.19.4
-	k8s.io/apiextensions-apiserver v0.19.2
+	k8s.io/apiextensions-apiserver v0.19.4
 	k8s.io/apimachinery v0.19.4
-	k8s.io/client-go v0.19.2
-	k8s.io/code-generator v0.19.2
-	k8s.io/component-base v0.19.2
+	k8s.io/client-go v0.19.4
+	k8s.io/code-generator v0.19.4
+	k8s.io/component-base v0.19.4
 	k8s.io/klog/v2 v2.4.0
-	k8s.io/kube-proxy v0.19.2
+	k8s.io/kube-proxy v0.19.4
+	k8s.io/kubernetes v1.19.4
 	k8s.io/utils v0.0.0-20200729134348-d5654de09c73
 	sigs.k8s.io/controller-runtime v0.6.3
 	sigs.k8s.io/structured-merge-diff/v4 v4.0.2 // indirect
 )
+
+replace (
+	github.com/onsi/ginkgo => github.com/onsi/ginkgo v1.11.0
```

Comment:
this shouldn't need to be done as a `replace`; just specify the correct version above

Reply:
the `apimachinery` [go.mod](https://github.com/openshift/kubernetes/blob/master/staging/src/k8s.io/apimachinery/go.mod#L24) needs to have that crazy incompatible ginkgo version. So this is just to get to making it work.